src/components/base.astro

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
```

Comment:
This component attempts to be the "theme editor" if you will. Meta tags, page title, colour scheme, and fonts are all defined here as the developer may wish.